packages/core/state/interaction/logics.ts

```diff
@@ -111,29 +115,17 @@ const downloadManifest = createLogic({
                     sort: sortColumn,
                 });
                 const count = await fileSet.fetchTotalCount();
-                const accumulator: { [index: string]: any } = {};
-                const selection: Selection = {
-                    filters: fileSet.filters.reduce((accum, filter) => {
-                        const existing = accum[filter.name] || [];
-                        return {
-                            ...accum,
-                            [filter.name]: [...existing, filter.value],
-                        };
-                    }, accumulator),
-                    indexRanges: [new NumericRange(0, count - 1).toJSON()],
-                    sort: sortColumn
-                        ? {
-                              annotationName: sortColumn.annotationName,
-                              ascending: sortColumn.order === SortOrder.ASC,
-                          }
-                        : undefined,
-                };
-                selections = [selection];
-            } else {
-                const fileSelection = selection.selectors.getFileSelection(state);
-                selections = fileSelection.toCompactSelectionList();
+                fileSelection = new FileSelection([
```

Comment:
I'm not entirely certain how to interpret these changes, but, I'm not getting expected behavior from this feature when I have a collection selected. How did you test this change?

Reply:
Ah this change isn't why this is failing, the CsvService wasn't pointing at the correct dataset API resource